### PR TITLE
Add metadata file for the new landing page.

### PR DESCRIPTION
**Pull request by kaushik-rishi**
metadata of 2 sections is present

- [x] categories of apps
- [x] how to use guide

### CODE REVIEW

**Comment by anilmurty** [comment]
how's it going with this? any blockers?

**Comment by kaushik-rishi** [comment]
> how's it going with this? any blockers?

Hi @anilmurty 
Just didn't want to get it merged before the #100 because I might need to change the deploy template for some requirement in the UI. So making another PR doesn't make sense.

I was a bit occupied, I'm back on working on the new landing page. :)

**Comment by anilmurty** [comment]
makes sense - thanks for the update.

**Comment by kaushik-rishi** [comment]
@anilmurty ready to merge, for just one tile, the custom import sdl tile on the categories section, i've used an in-memory object, because it had different configuration.

image
image

File: landing/metadata.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  "version": "0.0.1",
+  "categoriesTiles": {
+    "introText": "What would you like to do today?",
+    "tiles": [
+      {
+        "title": "Deploy a Blockchain Node",
+        "description": "Easy and low cost hosting for your blockchain nodes (RPC servers, Validators and more)",
+        "buttonText": "Choose a Template",
+        "route": "/landing/node-deployment",
+        "icon": "xrayView",
```
Comment (anilmurty) [inline landing/metadata.json:11]: file is named xray-view https://github.com/akash-network/deploy-templates/tree/main/template-icons/landing/icons - seems like either this file or the svg filename need to change for this to work

File: landing/metadata.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        "route": "",
+        "icon": "www",
+        "buttonEnabled": false
+      },
+      {
+        "title": "Deploy an AI/ ML Model",
+        "description": "Popular AI & ML models, deployed in  just a few clicks. Includes Stable Diffusion, GPT4All, Alpaca and more.",
+        "buttonText": "Choose a Template",
+        "route": "",
+        "icon": "electronicsChip",
```
Comment (anilmurty) [inline landing/metadata.json:29]: file is named electronic-chip https://github.com/akash-network/deploy-templates/tree/main/template-icons/landing/icons - seems like either this file or the svg filename need to change for this to work

File: landing/metadata.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        "buttonEnabled": false
+      }
```
Comment (anilmurty) [inline landing/metadata.json:32]: looks like we're missing a section for the custom SDL template here?

**Comment by anilmurty** [comment]
oh, ignore last comment (noted about using an "in-memory" object)

**Comment by anilmurty** [comment]
ignore last review comment (noted about using an "in-memory" object)

File: landing/metadata.json (hunk 1 above)
Comment (kaushik-rishi) [reply landing/metadata.json:11]: Hey Anil, i'll add the paths as well, but this is how i've referenced the SVG's for now.

https://github.com/kaushik-rishi/console/blob/feat-new-landing-page/web/src/components/Icons/index.tsx#L545-L562

https://github.com/kaushik-rishi/console/blob/feat-new-landing-page/web/src/components/Icons/index.tsx#L594-L598

Comment (kaushik-rishi) [reply landing/metadata.json:11]: Added the svg file paths as well

Comment (anilmurty) [reply landing/metadata.json:11]: oh, I didn't relize they were hard-coded. Was hoping we could allow them to be changed via the config files but we can refactor that later

**Comment by anilmurty** [comment]
I'll merge this now

**Review by anilmurty — APPROVED** [review]

**Pull request merged**